src/UI/Handlebars/Helpers/Artist.js

```diff
@@ -22,7 +22,7 @@ Handlebars.registerHelper('poster', function() {
 
 
 Handlebars.registerHelper('MBUrl', function() {
-    return 'https://musicbrainz.org/artist/' + this.mbId;
+    return 'https://musicbrainz.org/artist/' + this.foreignArtistId;
```

Comment:
Will look at reworking External link URLs to its own field going forward when we return multiple from metadata.